### PR TITLE
OP-779 Remove obsolete fields (ADM_OPE_ID_A, ADM_DATE_OP, ADM_RESOP)

### PR DESCRIPTION
**Pull request by mwithi**
See OP-779.

### CODE REVIEW

File: sql/step_04_all_following_steps.sql [hunk 1]
```diff
@@ -78,3 +78,4 @@ source step_81_link_bill_admission.sql;
 source step_82_anamnesis_patienthistory.sql;
 source step_83_fix_operationrow_column_names.sql;
 source step_84_session_table.sql;
+source step_XX_remove_old_admission_fields;
```
Comment (dbmalkovsky) [inline sql/step_04_all_following_steps.sql:81]:
```suggestion
source step_85_remove_old_admission_fields;
```

File: sql/step_XX_remove_old_admission_fields.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+ALTER TABLE OH_ADMISSION
```
Comment (dbmalkovsky) [inline sql/step_XX_remove_old_admission_fields.sql:1]: change `XX` to `85`

**Review by dbmalkovsky — APPROVED** [review]
LGTM with the renaming of the sql file

File: sql/step_04_all_following_steps.sql (hunk 1 above)
Comment (mwithi) [reply sql/step_04_all_following_steps.sql:81]: yes, I will do just before the merge, thanks!

File: sql/step_XX_remove_old_admission_fields.sql (hunk 2 above)
Comment (mwithi) [reply sql/step_XX_remove_old_admission_fields.sql:1]: yes, I will do just before the merge, thanks!

**Pull request merged**